### PR TITLE
Adds attributes to the repository object

### PR DESCRIPTION
**Pull request by timmo001**
Adds a few extra properties to `AIOGitHubAPIRepository` and adds two new classes, `AIOGitHubAPIRepositoryClones` and `AIOGitHubAPIRepositoryViews`

Relates to https://github.com/home-assistant/core/pull/38458

### CODE REVIEW

File: aiogithubapi/objects/repository/__init__.py [hunk 1]
```diff
@@ -58,14 +62,34 @@ def topics(self) -> None:
...
+    @property
+    def homepage(self) -> str:
+        return self.attributes.get("homepage")
+
+    @property
+    def stargazers(self) -> str:
+        return self.attributes.get("stargazers_count")
+
+    @property
+    def watchers(self) -> str:
```
Comment (ludeeus) [inline aiogithubapi/objects/repository/__init__.py:82]: the property name should match the key in the API response.

File: aiogithubapi/objects/repository/__init__.py [hunk 2]
```diff
@@ -58,14 +62,34 @@ def topics(self) -> None:
...
     @property
     def default_branch(self) -> None:
         return self.attributes.get("default_branch")
 
+    @property
+    def homepage(self) -> str:
+        return self.attributes.get("homepage")
+
+    @property
+    def stargazers(self) -> str:
```
Comment (ludeeus) [inline aiogithubapi/objects/repository/__init__.py:78]: the property name should match the key in the API response.

File: aiogithubapi/objects/repository/__init__.py [hunk 3]
```diff
@@ -58,14 +62,34 @@ def topics(self) -> None:
...
+    def watchers(self) -> str:
+        return self.attributes.get("watchers_count")
+
     @property
     def last_commit(self) -> None:
         return self._last_commit
 
+    @property
+    def owner(self) -> str:
+        return self.attributes.get("owner").get("login")
```
Comment (ludeeus) [inline aiogithubapi/objects/repository/__init__.py:91]: Can you reuse AIOGitHubAPIUser from aiogithubapi.objects.user here?

**Comment by timmo001** [comment]
Think I broke codecov :smile:

**Pull request merged**